e2e/node/e2e.test.ts

```diff
@@ -519,6 +524,30 @@ describe(`End-to-end access grant tests for environment [${environment}}]`, () =
       expect(updatedDatasetTtl).toBe(updatedDatasetAsOwnerTtl);
     });
 
+    it("can use the createContainerInContainer API to create a new container", async () => {
+      const containerNameSuggestion = "newTestContainer";
+      testContainerIriChild = `${testContainerIri}${containerNameSuggestion}/`;
```

Comment:
```suggestion
      testContainerIriChild = new URL(`${containerNameSuggestion}/`, testContainerIri).href;
```
We usually try to build URLs using the native URL object rather than string concatenation, which is more error-prone. It doesn't change much in this context, but it's a general guideline in the codebase :)